e2e/content-services/document-list/document-list-gallery.e2e.ts

```diff
@@ -0,0 +1,218 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ContentServicesPage } from '../../pages/adf/contentServicesPage';
+import { AcsUserModel } from '../../models/ACS/acsUserModel';
+import TestConfig = require('../../test.config');
+import resources = require('../../util/resources');
+import { LoginPage, StringUtil } from '@alfresco/adf-testing';
+import { AlfrescoApiCompatibility as AlfrescoApi } from '@alfresco/js-api';
+import { UploadActions } from '../../actions/ACS/upload.actions';
+import { FileModel } from '../../models/ACS/fileModel';
+import { NavigationBarPage } from '../../pages/adf/navigationBarPage';
+
+describe('Document List Component', () => {
+
+    const loginPage = new LoginPage();
+    const contentServicesPage = new ContentServicesPage();
+    let uploadedFolder, uploadedFolderExtra;
+    const uploadActions = new UploadActions();
+    let acsUser = null;
+    let testFileNode, pdfBFileNode;
+    const navBar = new NavigationBarPage();
+
+    beforeAll(() => {
+        this.alfrescoJsApi = new AlfrescoApi({
+            provider: 'ECM',
+            hostEcm: TestConfig.adf.url
+        });
+    });
+
+    afterEach(async (done) => {
```

Comment:
this afterEach section is not needed anymore
uploadedFolder, uploadedFolderExtra, testFileNode, pdfBFileNode are not created / used anywhere else in this file